### PR TITLE
new icon: networkx (original)

### PR DESCRIPTION
**Pull request by ozanuslan**
## original
networkx-original

## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] The PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

## Link to prove your SVG is correct and up-to-date
+ <a href="https://networkx.org/">Official Networkx Page</a>

## This PR adds/fixes
Addresses #898

### CODE REVIEW

File: icons/networkx/networkx-original.svg [hunk 1]
```diff
@@ -0,0 +1,2 @@
...
+
```
Comment (Panquesito7) [inline icons/networkx/networkx-original.svg:2]:
```suggestion
```

**Review by Panquesito7 — DISMISSED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by amino19 — DISMISSED** [review]
I checked for `plain-original`, it wouldn't exist at `networkx` artwork even.

BTW, Thanks for your contribution @ozanuslan, LGTM! :tada:

**Review by Thomas-Boi — CHANGES_REQUESTED** [review]
Hi @ozanuslan,

The bot detected that your icon contains strokes in them. Icomoon (the icon maker) would ignore these when converting them to font. This cause the font to miss some details:
image

Can you please [convert these strokes to fills](https://icomoon.io/#docs/stroke-to-fill) please? That will ensure the lines show up in the font version.

**Review by Panquesito7 — APPROVED** [review]

**Review by amino19 — APPROVED** [review]

**Comment by Thomas-Boi** [comment]
> Pardon the commit message, strokes have been converted to fills as requested. If strokes are not allowed perhaps a check can be added to the actions? 🤔

We actually do have a check. It's in the `peek-bot` message at the top. However, we can add it in `check-bot` as well.

**Review by Thomas-Boi — APPROVED** [review]

**Pull request merged**